Objective-C/CBLReplicator.mm

```diff
@@ -100,6 +99,7 @@ @implementation CBLReplicator
     CBLChangeNotifier<CBLReplicatorChange*>* _changeNotifier;
     CBLChangeNotifier<CBLDocumentReplication*>* _docReplicationNotifier;
     BOOL _resetCheckpoint;
+    BOOL _cancelSuspending;
```

Comment:
may be add a single line of comment, describing when this will be set and reset. or purpose of this.